source/jormungandr/tests/routing_tests_experimental.py

```diff
@@ -332,26 +332,78 @@ def test_max_mode_direct_path_duration(self):
         query = (query + '&max_{mode}_direct_path_duration={max_dp_duration}' + '&debug=true').format(
             mode=mode, max_dp_duration=direct_path_duration - 1
         )
+        response = self.query_region(query)
+        # New Default -> 'journeys' in response
+        # Distributed -> 'journeys' not in response
+        assert op('journeys' not in response)
+
+    return test_ko_direct_path_longer_than_max_mode_direct_path_duration
+
+
+@dataset({"main_routing_test": {"scenario": "distributed"}})
+class TestDistributedMaxDurationForDirectPathUpperLimit(NewDefaultScenarioAbstractTestFixture):
+    """
+    Test max_{mode}_direct_path_duration's upper limit
+
+    Direct path should be filtered if its duration is greater than max_{mode}_direct_path_duration
+    """
+
+    s = '8.98311981954709e-05;8.98311981954709e-05'
+    r = '0.0018864551621048887;0.0007186495855637672'
+    test_max_walking_direct_path_duration = _make_function_upper_limit(s, r, 'walking', operator.truth)
+    test_max_car_direct_path_duration = _make_function_upper_limit(s, r, 'car', operator.truth)
+    test_max_bss_direct_path_duration = _make_function_upper_limit(s, r, 'bss', operator.truth)
+    test_max_bike_direct_path_duration = _make_function_upper_limit(s, r, 'bike', operator.truth)
+
+    a = '0.001077974378345651;0.0007186495855637672'
+    b = '8.98311981954709e-05;0.0002694935945864127'
+    test_max_taxi_direct_path_duration = _make_function_upper_limit(a, b, 'taxi', operator.truth)
+
+
+def _make_function_lower_limit(from_coord, to_coord, mode):
```

Comment:
This is not a lower limit (min), this is just under the upper limit:
```suggestion
def _make_function_test_under_upper_limit(from_coord, to_coord, mode):
```